core/templates/dev/head/domain/exploration/SubtitledHtmlObjectFactorySpec.js

```diff
@@ -27,12 +27,13 @@ describe('SubtitledHtml object factory', function() {
       shof = $injector.get('SubtitledHtmlObjectFactory');
       subtitledHtml = shof.createFromBackendDict({
         html: '<p>some html</p>',
-        audio_translations: [{
-          language_code: 'en',
-          filename: 'a.mp3',
-          file_size_bytes: 20,
-          needs_update: false
-        }]
+        audio_translations: {
+          en: {
```

Comment:
Yeah, I think so :)

I would suggest we hold off on doing this for audio too unless we find that we need it. What do you think?